src/input/mpegts/satip/satip.c

```diff
@@ -384,12 +398,6 @@ satip_device_hack( satip_device_t *sd )
     sd->sd_pids_max    = 128;
     sd->sd_pids_len    = 2048;
     sd->sd_no_univ_lnb = 1;
-  } else if (strstr(sd->sd_info.manufacturer, "AVM Berlin") &&
```

Comment:
Well, the way I set it up now is that by default it enables safe defaults for FRITZ!-devices, but the settings are saved in the config files and the user can turn them on and off at will. I do not think having to modify and recompile sources just to fix the settings if a new firmware lands is the right way to go.